addons/sale_loyalty/models/sale_order.py

```diff
@@ -60,6 +60,7 @@ def action_confirm(self):
             if any(order._get_real_points_for_coupon(coupon) < 0 for coupon in all_coupons):
                 raise ValidationError(_('One or more rewards on the sale order is invalid. Please check them.'))
             order._update_programs_and_rewards()
+            order.validate_taxes_on_sales_order()
```

Comment:
Since `validate_taxes_on_sales_order` is already defined in `sale`, wouldn't it be better to call it once in `sale/models/sale_order.py > action_confirm` and remove the override of `action_confirm` in `sale_account_taxcloud` as well ?

That way, taxes won't have to be validated twice with requests to the TaxCloud API, and we will have less places where we have to manually manage it in sub-modules.

Reply:
You're right, I think this is better 👌🏾.
Thanks for the suggestion.